ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.service.ts

```diff
@@ -145,6 +148,13 @@ export class AddressInputService implements OnDestroy {
         distinctUntilChanged(),
         map(val => ({ isReadOnly: val }))),
 
+      this.defaultCountryCode$.pipe(
+          cachingCountryInfoOp,
+          map(countryInfo => ({ country: (countryInfo ? countryInfo.code : ''), countryInfo })),
+          map(countryInfoState => ({formData: new Address({country: countryInfoState.country}),
+            ...countryInfoState, formDataSource: 'INPUT' }))
+      ),
+
```

Comment:
Setting the defaultCountry is more or less the same as getting an address passed in, but with just the country.